library/ajax/pharmacy_autocomplete/index.php

```diff
@@ -0,0 +1,42 @@
+<?php 
+require_once('../../../interface/globals.php');
+
+
+?>
+
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Demo</title>
+  <link rel="stylesheet" href="../../js/jquery-ui.min.css" type="text/css" /> 
+
+</head>
+<body> 
+
+	<form action='' method='post'>
+		<p><label>Pharmacy Names : </label><input type='text' name='names' id="pharm" value='' class='auto'></p>
+		<p><label>City : </label><input type='text' name='city' id="city" value='' placeholder='Enter City First'></p>
```

Comment:
translate two items in above line:
``````
<p><label><?php echo xlt('City'); ?> : </label><input type='text' name='city' id="city" value='' placeholder='<?php xla('Enter City First'); ?>></p>